benchmarks/synchrobench/src/main/java/com/oath/oak/synchrobench/maps/OakMap.java

```diff
@@ -0,0 +1,120 @@
+package com.oath.oak.synchrobench.maps;
+
+
+import com.oath.oak.*;
+import com.oath.oak.synchrobench.contention.abstractions.CompositionalOakMap;
+import com.oath.oak.synchrobench.contention.benchmark.Parameters;
+
+public class OakMap<K, V> implements CompositionalOakMap<K, V> {
+    private com.oath.oak.OakMap<MyBuffer, MyBuffer> oak;
+    private OakMapBuilder<MyBuffer, MyBuffer> builder;
+    private MyBuffer minKey;
+    private OakBufferView oakView;
+
+    public OakMap() {
+        minKey = new MyBuffer(Integer.BYTES);
+        minKey.buffer.putInt(0, Integer.MIN_VALUE);
+        builder = new OakMapBuilder<MyBuffer, MyBuffer>()
+                .setKeySerializer(MyBufferOak.serializer)
+                .setValueSerializer(MyBufferOak.serializer)
+                .setMinKey(minKey)
+                .setComparator(MyBufferOak.keysComparator)
+                .setChunkBytesPerItem(Parameters.keySize)
+                .setChunkMaxItems(Chunk.MAX_ITEMS_DEFAULT);
+        oak = builder.build();
+        oakView = oak.createBufferView();
+    }
+
+    @Override
+    public boolean getOak(K key) {
+        return oak.get((MyBuffer) key) != null;
+    }
+
+    @Override
+    public void putOak(K key, V value) {
+        oak.put((MyBuffer) key, (MyBuffer) value);
+    }
+
+    @Override
+    public boolean putIfAbsentOak(K key, V value) {
+        return oak.putIfAbsent((MyBuffer) key, (MyBuffer) value);
+    }
+
+    @Override
+    public void removeOak(K key) {
+        oak.remove((MyBuffer) key);
+    }
+
+    @Override
+    public boolean computeIfPresentOak(K key) {
+        return false;
+    }
+
+    @Override
+    public void computeOak(K key) {
+
+    }
+
+    @Override
+    public boolean ascendOak(K from, int length) {
+        int i;
+        try (com.oath.oak.OakMap<MyBuffer, MyBuffer> sub = oak.tailMap((MyBuffer) from, true)) {
+            OakBufferView<MyBuffer> oakView = sub.createBufferView();
+            i = 0;
+            OakIterator<OakRBuffer> iter = oakView.keysIterator();
```

Comment:
We should probably talk about this with the team. I think its not fair we are returning bufferview and not deserializing. Maybe we can have a command line option to use this or that and see the performance?

Reply:
I agree. Suggest we open an issue and get to it in a different PR